h2o-automl/src/main/java/ai/h2o/automl/TargetEncoder.java

```diff
@@ -176,6 +177,19 @@ public Frame transformBinaryTargetColumn(Frame data, int targetIndex)  {
         return res;
     }
 
+    public Frame appendBinaryTargetColumn(Frame data, int targetIndex)  {
```

Comment:
What is this method supposed to do?

Reply:
@michalkurka this one was added just for the test. Should not have added it to the code..but It has usages from test and it is not abandoned piece of code. This method is actually a copy of the method right above `transformBinaryTargetColumn` with the exception that it does not mutate column in-place but append another one .... I was investigating the case when binary label encoding is performed wrongly depending on what label is coming first in the dataset. In the codebase I saw a comments that domain[1] should be always stand for a target class.... but I'm now sure if someone is taking care of it automatically. My test shows that encoding will be different. But I need to test it in R.